packages/core/src/resolver-loader.ts

```diff
@@ -28,7 +28,21 @@ export class ResolverLoader {
     if (!this.#resolver) {
       let config: Options;
       if (existsSync(this.#configFile)) {
-        config = readJSONSync(this.#configFile);
```

Comment:
This type inference here is a complete lie. The right hand side is a JSON with a bunch of `any` types and the config has the ability to have RegEx objects 🙈 i don't know what the right thing to do here is to deserialise this properly but I deferred the type inference while I added the checks I needed